MediaBrowser.Api/Images/ImageService.cs

```diff
@@ -468,9 +470,18 @@ public void Delete(DeleteUserImage request)
             var userId = request.Id;
             AssertCanUpdateUser(_authContext, _userManager, userId, true);
 
-            var item = _userManager.GetUserById(userId);
+            var user = _userManager.GetUserById(userId);
+            try
+            {
+                File.Delete(user.ProfileImage.Path);
+            }
+            catch (IOException e)
+            {
+                Logger.LogError(e, "Error deleting user profile image:");
+            }
 
-            item.DeleteImage(request.Type, request.Index ?? 0);
+            user.ProfileImage = null;
+            _userManager.UpdateUser(user);
```

Comment:
How do we want to handle when there is an error in deleting a profile image. Currently, we just leave it on disk and pretend it's not there anymore, but that could cause problems if the user wants to upload an image with the same extension later. Another solution would be to just not delete the image and have the API return some type of error.

Reply:
Id say for now, have it overwrite the image on a new upload so it doesn't matter and doesn't change anything. Long term, we should send an error back but that's an API breaking change now.